WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+final class BlazeCampaignsViewController: UIViewController {
+
+    // MARK: - Views
+
+    private lazy var dismissButton: UIBarButtonItem = {
```

Comment:
```suggestion
    private lazy var plusButton: UIBarButtonItem = {
```

I assume this was overlooked 😄

Reply:
Good catch! Fixed